xrmocap/model/registrant/smplify.py

```diff
@@ -637,6 +705,28 @@ def __compute_loss__(self,
 
         return losses
 
+    def __post_process_loss__(self, losses: dict, **kwargs) -> dict:
+        """Process losses and map the losses to respective parameters.
+
+        Args:
+            losses (dict): Original loss, use handler_key as keys.
+
+        Returns:
+            dict: Processed loss, use parameter names as keys.
+                Original keys included.
+        """
+
+        for loss_key in losses.copy().keys():
```

Comment:
```suggestion
        for loss_key in losses.keys():
```

Reply:
Fixed in [cab8731](https://github.com/openxrlab/xrmocap/pull/56/commits/cab87319d90d6f7f6e5f611602f0b4c4d7d6f938).